perfkitbenchmarker/linux_benchmarks/cloudharmony_network_benchmark.py

```diff
@@ -315,6 +382,10 @@ def _Run(benchmark_spec, test):
     metadata['throughput_time'] = True
   if FLAGS.ch_network_throughput_slowest_thread:
     metadata['throughput_slowest_thread'] = True
+  if FLAGS.ch_network_test == 'dns':
+    metadata['dns_recursive'] = True
```

Comment:
what is this meatadata mean?

Reply:
This performs a recursive rather than an authoritative DNS query.
I think the DNS server that is being queried is not an authoritative server, so it throws an error unless this cloudharmony flag is set.